setupext.py

```diff
@@ -645,9 +655,10 @@ def do_custom_build(self, env):
             subprocess.check_call([make], env=env, cwd=src_path)
         else:  # compilation on windows
             shutil.rmtree(src_path / "objs", ignore_errors=True)
+            is_x64 = platform.architecture()[0] == '64bit'
             msbuild_platform = (
-                'x64' if platform.architecture()[0] == '64bit' else 'Win32')
-            base_path = Path("build/freetype-2.6.1/builds/windows")
+                'ARM64' if platform.machine() == 'ARM64' else 'x64' if is_x64 else 'Win32')
+            base_path = Path("build/freetype-{}/builds/windows".format(LOCAL_FREETYPE_VERSION))
```

Comment:
Mild preference for fstring, but fine either way.

```suggestion
            base_path = Path(f"build/freetype-{LOCAL_FREETYPE_VERSION}/builds/windows")
```

Reply:
Fixed on last commit